### PR TITLE
Add missing BatchNormalization include for ApplyAlpha layer

### PR DESCRIPTION
**Pull request by thesps**
As it says: BatchNormalization includes weren't registered for ApplyAlpha layer, so a model using quantizer with `alpha`, and no BatchNormalization layers wouldn't compile.

### CODE REVIEW

**Comment by thesps** [comment]
I've added the expression-balanced summation to Softmax. This is required to maintain unrolled inference when rounding or saturation mode are used, which I found improve the performance.

File: hls4ml/templates/vivado/nnet_utils/nnet_activation.h [hunk 1]
```diff
@@ -226,36 +223,38 @@ void softmax(data_T data[CONFIG_T::n_in], res_T res[CONFIG_T::n_in]){
     bool initialized = false;
     typename CONFIG_T::exp_table_t exp_table[CONFIG_T::table_size];
     typename CONFIG_T::inv_table_t invert_table[CONFIG_T::table_size];
+	#pragma HLS resource variable=exp_table core=ROM_nP_BRAM
```
Comment (vloncar) [inline hls4ml/templates/vivado/nnet_utils/nnet_activation.h]: Are you happy with this being hardcoded since it is not configurable?

Also, spaces instead of tabs.

File: hls4ml/templates/vivado/nnet_utils/nnet_activation.h [hunk 2]
```diff
@@ -226,36 +223,38 @@ void softmax(data_T data[CONFIG_T::n_in], res_T res[CONFIG_T::n_in]){
...
 #endif
     if (!initialized) {
-        // Note we are exponentiating the inputs, which have type data_T
-        init_exp_table<data_T, CONFIG_T>(exp_table);
-        // Note we are inverting the exponentials, which have type exp_table_t
-        init_invert_table<typename CONFIG_T::exp_table_t, CONFIG_T>(invert_table);
+		// Note we are exponentiating the inputs, which have type data_T
+		init_exp_table<data_T, CONFIG_T>(exp_table);
+		// Note we are inverting the exponentials, which have type exp_table_t
+		init_invert_table<typename CONFIG_T::exp_table_t, CONFIG_T>(invert_table);
```
Comment (vloncar) [inline hls4ml/templates/vivado/nnet_utils/nnet_activation.h]: Spaces instead of tabs please.

**Review by vloncar — CHANGES_REQUESTED** [review]

File: hls4ml/templates/vivado/nnet_utils/nnet_activation.h (hunk 1 above)
Comment (thesps) [reply hls4ml/templates/vivado/nnet_utils/nnet_activation.h]: Whoooooops that pragma was not needed anymore.

File: hls4ml/templates/vivado/nnet_utils/nnet_activation.h (hunk 2 above)
Comment (thesps) [reply hls4ml/templates/vivado/nnet_utils/nnet_activation.h]: One of those rare occasions I used the editor in Vivado GUI.
